neuralmagic/benchmarks/scripts/backend_request_func.py

```diff
@@ -43,6 +43,12 @@ def trim_prefix(text: str, prefix_str: str) -> str:
     return text[len(prefix_str):]
 
 
+def trim_suffix(text: str, suffix_str: str) -> str:
+    assert len(text) >= len(suffix_str)
+    assert text[-1 * len(suffix_str):] == suffix_str
+    return text[:-1 * len(suffix_str)]
```

Comment:
nit: why does this require that the suffix exists and fail if it doesn't? i would simplify it to this if the assertion isn't required
```suggestion
def trim_suffix(text: str, suffix_str: str) -> str:
    if text.endswith(suffix_str):
        return text[:-len(suffix_str)]
    return text
```

Reply:
It is out of the timing code and I think is good to keep around so we know we aren't trimming any actual data.